src/inference/include/openvino/runtime/core.hpp

```diff
@@ -672,4 +672,10 @@ class OPENVINO_RUNTIME_API Core {
     RemoteContext get_default_context(const std::string& device_name);
 };
 
+/**
+ * @brief Shut down the openvino by releasing dependent resources.
```

Comment:
Can we extend description why we need to have this function.
Example: https://github.com/protocolbuffers/protobuf/blob/520c601c99012101c816b6ccc89e8d6fc28fdbb8/src/google/protobuf/message_lite.h#L570